docs/installation.rst

```diff
@@ -223,14 +208,11 @@ the libev event loop by doing the following:
 
 .. code-block:: python
 
-    >>> from cassandra.io.libevreactor import LibevConnection
```

Comment:
again only the external name of the package is changing the to code itself isn't changing any names.